tests/spcs/test_services.py

```diff
@@ -327,3 +328,177 @@ def test_list_endpoints_cli(mock_list_endpoints, mock_cursor, runner):
+    min_instances = 2
+    max_instances = 3
+    query_warehouse = "test_warehouse"
+    auto_resume = False
+    comment = "this is a test"
+    result = runner.invoke(
+        [
+            "spcs",
+            "service",
+            "set",
+            service_name,
+            "--min-instances",
+            str(min_instances),
+            "--max-instances",
+            str(max_instances),
+            "--query-warehouse",
+            query_warehouse,
+            "--no-auto-resume",
+            "--comment",
+            comment,
+        ]
+    )
+    mock_set.assert_called_once_with(
+        service_name=service_name,
+        min_instances=min_instances,
+        max_instances=max_instances,
+        query_warehouse=query_warehouse,
+        auto_resume=auto_resume,
+        comment=to_string_literal(comment),
+    )
+    assert result.exit_code == 0, result.output
+    assert "Statement executed successfully" in result.output
+
+
+@patch("snowflake.cli.plugins.spcs.services.manager.ServiceManager.set_property")
+def test_set_property_no_properties_cli(mock_set, runner):
+    service_name = "test_service"
+    mock_set.side_effect = NoPropertiesProvidedError(
+        f"No properties specified for service '{service_name}'. Please provide at least one property to set."
+    )
+    result = runner.invoke(["spcs", "service", "set", service_name])
+    assert result.exit_code == 1, result.output
+    assert "No properties specified" in result.output
+    mock_set.assert_called_once_with(
+        service_name=service_name,
+        min_instances=None,
+        max_instances=None,
+        query_warehouse=None,
+        auto_resume=None,
+        comment=None,
+    )
+
+
+@patch(
+    "snowflake.cli.plugins.spcs.services.manager.ServiceManager._execute_schema_query"
+)
+def test_unset_property(mock_execute_query):
+    service_name = "test_service"
+    cursor = Mock(spec=SnowflakeCursor)
+    mock_execute_query.return_value = cursor
+    result = ServiceManager().unset_property(service_name, True, True, True, True, True)
+    expected_query = "alter service test_service unset min_instances,max_instances,query_warehouse,auto_resume,comment"
+    mock_execute_query.assert_called_once_with(expected_query)
+    assert result == cursor
+
+
+def test_unset_property_no_properties():
+    service_name = "test_service"
+    with pytest.raises(NoPropertiesProvidedError) as e:
+        ServiceManager().unset_property(service_name, False, False, False, False, False)
+    assert (
+        e.value.message
+        == f"No properties specified for service '{service_name}'. Please provide at least one property to reset to its default value."
+    )
+
+
+@patch("snowflake.cli.plugins.spcs.services.manager.ServiceManager.unset_property")
+def test_unset_property_cli(mock_unset, mock_statement_success, runner):
+    cursor = mock_statement_success()
+    mock_unset.return_value = cursor
+    service_name = "test_service"
+    result = runner.invoke(
```

Comment:
Do we have a way to disallow customer setting property value in unset command?
Could we add a test to verify `["spcs", "service", "unset", service_name, "--min-instances", 1]` is not allowed?